src/types/index.ts

```diff
@@ -0,0 +1,15 @@
+export type Response<T> = {
+  data: T;
+  message: string;
+};
+
+export type User = {
+  name: string;
+  email: string;
+  role: string;
+};
+
+export type AuthState = {
+  user: User;
+  token: string;
+};
```

Comment:
I'm not sure when you will use this file, if for redux, better to move types to redux, its will be hard to support this file over time

Reply:
moved types to redux in this commit https://github.com/ZenBit-Tech/faq_team_fe/pull/1/commits/6946b1fe06bc34499ad0193d2c5396d51a17d410